src/seedu/addressbook/data/person/Address.java

```diff
@@ -20,13 +20,70 @@
      *
      * @throws IllegalValueException if given address string is invalid.
      */
+    private class Block {
+    	private final String value;
+    	
+    	public Block(String value) {
+    		this.value=value;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.